partner_create_by_vat/models/res_partner.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+
+import requests
+
+from string import maketrans
+from stdnum.eu.vat import check_vies
```

Comment:
Yes, but the problem is another: the web client loads every module in the addons paths if it has a static directory, even if they're not installed. When whole repository branches are added to the addons path, which is not uncommon, you don't want to be installing all the python imports for modules you are not using.

You have the external dependency correctly noted in the manifest file, so you can safely move the import to the method where it does no harm anymore. This is a common OCA convention.